src/Select.js

```diff
@@ -1066,6 +1072,51 @@ class Select extends React.Component {
 		);
 	}
 
+	renderUniqueSelect(className, valueArray) {
+		const focusedOptionIndex = this.getFocusableOptionIndex(valueArray[0]);
```

Comment:
This line till 1084 are the same as the fifth lines in `render`, can't we just pass `focusedOption` as a param?